code/modules/mafia/abilities/investigative/investigate.dm

```diff
@@ -0,0 +1,27 @@
+/**
+ * Investigate
+ *
+ * During the night, Investigating will reveal the person's faction.
+ */
+/datum/mafia_ability/investigate
+	name = "Investigate"
+	ability_action = "investigate"
+
+/datum/mafia_ability/investigate/perform_action_target(datum/mafia_controller/game, datum/mafia_role/day_target)
+	. = ..()
+	if(!.)
+		return .
```

Comment:
i'd rather you keep putting `return FALSE` here and `return TRUE` at the end of the proc rather than keep relying on funny dot operator so all of this is exponentially easier to understand/is more explicitly stated